app/lib/controlled_work_mappings/cw1.yml

```diff
@@ -159,14 +159,16 @@
 - name: Other property 1_2
   type: text
   source: from_attribute
-  attribute: additional_non_smod_properties_value
+  attribute: additional_properties_value
```

Comment:
This makes me uncomfortable. I don't want the value of additional properties here, I want the value of additional _non-smod_ properties here. Like how further up in the file I want the value of additional _smod_ properties.